Python/Templates/Uwp/ProjectTemplates/Python/Windows IoT Core/BackgroundService/PythonBackgroundService.pyproj

```diff
@@ -51,10 +51,11 @@
   <PropertyGroup>
     <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
     <PtvsVersion Condition="'$(PtvsVersion)' == ''">3.0</PtvsVersion>
-    <PtvsTargetsFile>$(LocalAppData)\Microsoft\VisualStudio\$(VisualStudioVersion)Exp\Extensions\Microsoft Corporation\Python Tools for Visual Studio - UWP\$(PtvsVersion)\Microsoft.PythonTools.Uwp.targets</PtvsTargetsFile>
-    <PtvsTargetsFile Condition="'$(PtvsTargetsFile)' == '' or !Exists('$(PtvsTargetsFile)')">$(MSBuildProgramFiles32)\Microsoft Visual Studio $(VisualStudioVersion)\Common7\IDE\Extensions\Microsoft Corporation\Python Tools for Visual Studio - UWP\$(PtvsVersion)\Microsoft.PythonTools.Uwp.targets</PtvsTargetsFile>
+    <_PythonToolsPath Condition="!Exists($(_PythonToolsPath))">$([MSBuild]::GetRegistryValueFromView('HKEY_LOCAL_MACHINE\Software\Microsoft\PythonTools\$(VisualStudioVersion)', 'InstallDir', null, RegistryView.Registry32))</_PythonToolsPath>
+    <_PythonToolsPath Condition="!Exists($(_PythonToolsPath))">$(LocalAppData)\Microsoft\VisualStudio\$(VisualStudioVersion)Exp\Extensions\Microsoft Corporation\Python Tools for Visual Studio\$(PtvsVersion)</_PythonToolsPath>
+    <UwpTargetsFile>$(_PythonToolsPath)\..\..\Python Tools for Visual Studio - UWP\$(PtvsVersion)\Microsoft.PythonTools.Uwp.targets</UwpTargetsFile>
   </PropertyGroup>
 
-  <Import Project="$(PtvsTargetsFile)" />
+  <Import Project="$(UwpTargetsFile)" />
```

Comment:
To work with dev15, this will need to be:
```
<Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\Python Tools\Microsoft.PythonTools.Uwp.targets" />
```

Using intermediate variables is not supported.

You can clone `Python\Product\BuildTasks\TestTargets\Microsoft.PythonTools.targets` to `Microsoft.PythonTools.Uwp.targets` and update it to do the import correctly, then run `Python\Prerequisites\deploy_test_targets.cmd` on your dev machine to make imports work. The project templates should be as clean as possible.